app/models/project.rb

```diff
@@ -13,6 +13,7 @@ class Project < ActiveRecord::Base
 
   EXPERT_ROLES = [:expert, :owner]
 
+  belongs_to :default_tutorial, class_name: "Tutorial"
```

Comment:
Up to you man. You're going to be maintaining it. I thought I answered the question in my post above, that I think it makes sense that the project holds the reference to its default tutorial, but I could see it either way.